### PR TITLE
Change/add additional client details

### PR DESCRIPTION
**Pull request by jjeroch**

### CODE REVIEW

File: postgres/init/22-portal-content.sql [hunk 1]
```diff
@@ -86,6 +86,8 @@ aa0a0000-7fbc-1f2f-817f-bce0502c1015	1	2022-09-18 00:00:00+00	2022-09-18 00:00:0
...
 aa0a0000-7fbc-1f2f-817f-bce0502c1017	4	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Terms & Conditions Service Provider	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
 aa0a0000-7fbc-1f2f-817f-bce0502c1018	4	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Data Sharing Approval - allow CX to submit company data (company name, requester) to process the subscription	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
+aa0a0000-7fbc-1f2f-817f-bce0502c1090	1	2022-01-01 00:00:00.388+00	2022-01-01 00:00:00.388+00	\N	I confirm that my company has successfully received a Catena-X onboarding certificate for application providers / service providers issued by an official Catena-X Conformity Assessment Body (CAB). I acknowledge that I am able to upload the certificate in the subsequent registration process steps. For more information on conformity assessment please visit the association homepage (<- Link ).	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
```
Comment (Phil91) [inline postgres/init/22-portal-content.sql:89]: The name of the agreement is to long, currently we have a limit to 255 chars. You either need to shorten the name or we need to adjust the max column length

File: postgres/init/22-portal-content.sql [hunk 2]
```diff
@@ -86,6 +86,8 @@ aa0a0000-7fbc-1f2f-817f-bce0502c1015	1	2022-09-18 00:00:00+00	2022-09-18 00:00:0
...
+aa0a0000-7fbc-1f2f-817f-bce0502c1090	2	2022-01-01 00:00:00.388+00	2022-01-01 00:00:00.388+00	\N	I confirm that my company has successfully received a Catena-X onboarding certificate for application providers / service providers issued by an official Catena-X Conformity Assessment Body (CAB). I acknowledge that I am able to upload the certificate in the subsequent registration process steps. For more information on conformity assessment please visit the association homepage (<- Link ).	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
```
Comment (Phil91) [inline postgres/init/22-portal-content.sql:90]: same as above

**Review by Phil91 — CHANGES_REQUESTED** [review]

File: postgres/init/22-portal-content.sql (hunk 2 above)
Comment (jjeroch) [reply postgres/init/22-portal-content.sql]: done

File: postgres/init/22-portal-content.sql [hunk 3]
```diff
@@ -420,6 +426,8 @@ ed4116b0-f4bc-451e-aaa9-5df08969e1b4	4ba33360-72e6-4a56-a253-59761d694d2c	\N	htt
 7f9888cb-2f78-4f71-b7dd-a9231888aa55	4d44eb62-7ac8-4a4a-a0d0-fc3daca5edd5	\N	\N	\N
 3d8cb906-76a3-4cd7-ae99-4f9225437a27	11fd3bb9-6922-4b36-bc28-d30aded20231	\N	https://dtc-translator.adac.openresearch.com	\N
 37241577-3730-4d39-ab5b-3555a0d11c42	1ab6c7c9-ff68-47da-aad4-b88d83b757cd	7e828901-27cb-4078-bd15-00efd247c56a	https://fleet-management.adac.openresearch.com	\N
+ac6413b0-b179-462a-9074-e7d0bcc5ee5a	1d28a224-6293-4148-8b60-6714a0f1465d	476b0600-965b-4f10-8eb5-e4568859a889	https://catena-x.palantirfoundry.com/	\N
```
Comment (jjeroch) [inline postgres/init/22-portal-content.sql:429]: @Phil91 i think this is a duplicate now - I would delete the old row.  Do you agree?

Comment (Phil91) [reply postgres/init/22-portal-content.sql:429]: oh sorry, than i didn't look right while merging, thanks for double checking.
I agree can be deleted

File: postgres/init/22-portal-content.sql [hunk 4]
```diff
@@ -86,6 +86,8 @@ aa0a0000-7fbc-1f2f-817f-bce0502c1015	1	2022-09-18 00:00:00+00	2022-09-18 00:00:0
 aa0a0000-7fbc-1f2f-817f-bce0502c1016	2	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Marketplace Terms & Conditions	ac861325-bc54-4583-bcdc-9e9f2a38ff84	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
 aa0a0000-7fbc-1f2f-817f-bce0502c1017	4	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Terms & Conditions Service Provider	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
 aa0a0000-7fbc-1f2f-817f-bce0502c1018	4	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Data Sharing Approval - allow CX to submit company data (company name, requester) to process the subscription	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
+aa0a0000-7fbc-1f2f-817f-bce0502c1090	1	2022-01-01 00:00:00.388+00	2022-01-01 00:00:00.388+00	\N	I confirm that my company has successfully received a Catena-X onboarding certificate issued by an official Conformity Assessment Body (CAB). I awknowledge to upload the certificate. For more information please visit the association homepage (Link)  2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
```
Comment (Phil91) [inline postgres/init/22-portal-content.sql:89]: The text is still to long, max allowed would be:

_I confirm that my company has successfully received a Catena-X onboarding certificate for application providers / service providers issued by an official Catena-X Conformity Assessment Body (CAB). I acknowledge that I am able to upload the certificate in _

**Review by Phil91 — CHANGES_REQUESTED** [review]

File: postgres/init/22-portal-content.sql (hunk 4 above)
Comment (jjeroch) [reply postgres/init/22-portal-content.sql:89]: @Phil91 did you received that error with an upload test? asking cause I have tested the sentence earlier and 255 alphanumeric characters are allowed and the agreement name has 248 characters.
I have also been able to successfully load the name inside the dev db, I am surprised that it fails for you

Comment (jjeroch) [reply postgres/init/22-portal-content.sql:89]: ah maybe it was a space/tab issue - looks now better I believe

**Review by Phil91 — APPROVED** [review]

**Pull request merged**